ci/upload-launch.sh

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env bash
+
+set -e -u -o -x pipefail
+
+cd "$(dirname "$0")/../"
+
+PREFIX="playground"
+
+ci/build-test.sh
+
+source ".shared-ci/scripts/profiling.sh"
+
+.shared-ci/scripts/upload-assemblies.sh ${PREFIX}
+
+markStartOfBlock "Launching deployment"
+
+# Get first 8 characters of current git hash.
+GIT_HASH="$(git rev-parse HEAD | cut -c1-8)"
+
+spatial cloud launch "${PREFIX}_${GIT_HASH}" cloud_launch.json "${PREFIX}_${GIT_HASH}" --snapshot=snapshots/default.snapshot
```

Comment:
In retrospect - pulling the prefix + git hash out to a shared function may be preferable. Given that its used here & upload-assemblies.

Reply:
Yup, having it used in both scripts without clear dependency looks quite fragile. Can we instead just pass the full name to upload-assemblies?